pom.xml

```diff
@@ -61,8 +61,17 @@
                         <encoding>${project.build.sourceEncoding}</encoding>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-site-plugin</artifactId>
+                    <version>3.7.1</version>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-project-info-reports-plugin</artifactId>
+                    <version>3.0.0</version>
+                </plugin>
```

Comment:
I think the project can work without this

Reply:
I add that to fix this:
![image](https://user-images.githubusercontent.com/105455921/197610307-60bd3237-94b5-4e19-8ae3-dde23a52ec8d.png)

Reply:
Why do you need to run 'site'? 'package' is enough.

Reply:
I don`t know which tests I need to run and which I don`t need to run. I did not installed maven until today(missed this step). I tried to use mvn clean package command and it wasn`t working until today. I used maven tool. 
I installed maven. It`s working in terminal now.